README.md

```diff
@@ -1,57 +1,116 @@
 - Keep everything compatible with deployment to OpenShift.
-- Reduce technical debt as much as possible and keep cleaning it up as you go.
+- Minimize technical debt.
 
 ## Requirements:
-- Git
-- “Make” (installed by default on OS X and Linux)
+
+- [Git](https://git-scm.com)
+- [make](http://man7.org/linux/man-pages/man1/make.1.html) (installed by default on OS X and Linux)
 - [Docker](https://www.docker.com/)
-- OpenShift CLI
-- Docker Compose (included with Docker on Windows and Mac; only for convenience when running locally)
+- [OpenShift CLI](https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html#cli-reference-get-started-cli)
+- Docker Compose (included with [Docker](https://docs.docker.com/install/) on Windows and Mac; used to run locally)
 
 ## Getting Started
-The application is fully Dockerized, which makes deployment to OpenShift possible. An additional benefit is that the application can easily be run locally as a Docker network using Docker-Compose. If someone prefers to run Python, Node and PostgreSQL directly on their workstation as individual services that is fully possible, but the seamless experience described here lends itself to easy setup and switching between branches. Not that the Docker-Compose commands are not intended to interact in any way with OpenShift (see "Deployment" below).
 
-1. Install [Docker](https://www.docker.com/) (if you haven't already)
-1. Checkout the code: git clone https://github.com/bcgov/openshift-launchpad.git (or use ssh)
-1. Under the root directory, run `make run` in your terminal
-1. Visit the application on the following ports/URLs:
-	- The client container exposes port 3000 and can be viewed by visiting http://localhost:3000
-	- The server container exposes port 5000 and can be viewed by visiting http://localhost:5000
-	- The database container exposes port 5435 and can be viewed by using pgAdmin, postico, or psql
+The application is fully containerized, simplifying both local development and deployment to OpenShift. The processes described in this document use Docker Compose to run services locally. Note that running the Python, Node, and PostgreSQL services locally without Docker containers is possible, but is left as an excersise for the reader. Further, note that the Docker Compose commands are not intended to interact in any way with OpenShift (see "Deployment" below).
+
+### Local Deployment
+
+1. Install [Docker](https://www.docker.com/)
+2. Checkout the project `git clone https://github.com/bcgov/openshift-launchpad.git`
+3. Under the root directory, run `make run` in your terminal
+4. Inspect the application services exposed at the following URLs.
+	- The client container exposes port 3000 locally at http://localhost:3000
+	- The server container exposes port 5000 locally at http://localhost:5000
+	- The database container exposes port 5435 and can be inspected using pgAdmin, Postico, psql, etc.
```

Comment:
@ashtonmeuser maybe it's better to include something about server-migrate runs database migrations